### PR TITLE
Feature agglomeration bugfix

### PR DESCRIPTION
**Pull request by LaSerpe**
This pull request solves issue #276 
The variable Parent_CV had no initial value causing undefined behaviour during multigrid agglomeration process.

The variable is now initialized to 0 in CPoint constructor in dual_geometry_structure.cpp.

### CODE REVIEW

File: Common/src/dual_grid_structure.cpp [hunk 1]
```diff
@@ -59,7 +59,7 @@ CPoint::CPoint(unsigned short val_nDim, unsigned long val_globalindex, CConfig *
 	Coord = new su2double[nDim];
 
 	/*--- Indicator if the control volume has been agglomerated ---*/
-  
+    Parent_CV = 0;
```
Comment (economon) [inline Common/src/dual_grid_structure.cpp]: Can you please double check that the other constructors for CPoint are ok too? There are 3 of them due to overloading, I believe.

Also, small thing, but can you please take the opportunity to clean up the indentation in this routine (and others) of this file? We can make a big impact by cleaning small pieces of the code as we go. The standard is two spaces (no tab characters).

Comment (LaSerpe) [inline Common/src/dual_grid_structure.cpp]: Yep, I forgot the other 2 constructors..I just fixed them and indentation should be ok as well.
Let me know if we can further improve  the style.

**Comment by talbring** [comment]
So does this solve the random behavior of the flat plate test case ?

**Comment by LaSerpe** [comment]
Hard to tell if it solves the issue once and for all since that random behaviour was pretty rare.
After initializing those values to zero I was no longer able to reproduce the issue on my machine. 
By the way, on another branch I have (feature_driver_prototype) this issue was always occurring, after this modification it disappeared in that branch too, so I'm pretty confident we fixed it.

As far as I know, among all the reg tests, that random behaviour was affecting only the flat plate case.
Are you aware of any other test I can check that are showing this issue?

**Comment by LaSerpe** [comment]
merging in now.. Thanks everybody

**Pull request merged**